web/src/styles/base/gap.scss

```diff
@@ -0,0 +1,23 @@
+.gap-0 {
+  gap: 0px;
+}
+
+.gap-1 {
+  gap: 0.5rem;
+}
+
+.gap-2 {
+  gap: 1rem;
+}
+
+.gap-3 {
+  gap: 1.5rem;
+}
+
+.gap-4 {
+  gap: 2rem;
+}
+
+.gap-5 {
+  gap: 2.5rem;
+}
```

Comment:
[dust] Are these consistent with USWDS naming conventions? Typically their increments are -1, -105, -2, -205, etc.

Reply:
I decided to just do a loose implementation of more or less what I needed instead of a more complete implementation just for a bit of time saving